Week-11/exercise11.2/step-counter-app/src/reducer.js

```diff
@@ -0,0 +1,20 @@
+let initialState = { counter: 0 };
+
+function stepCountReducer(currentState=initialState, action){
+    console.log(action)
+    console.log(currentState)
+    switch (action.type){
+        case "INCREASE_COUNT":{
+            return { counter: action.counter + 1 };
+        }
+        case "DECREASE_COUNT":{
+            return { counter: action.counter - 1 };
+        }
```

Comment:
use the currentState while making incremental modifications in the state. { counter: currentState.counter + 1 }